### PR TITLE
Make a basic User model (Including secure passwords)

### PR DESCRIPTION
**Pull request by hoanganhh**

### CODE REVIEW

**Comment by hoanganhh** [comment]
ready

**Comment by hoanganhh** [comment]
ready

**Comment by hoanganhh** [comment]
ready

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,2 @@
-<% provide(:title, "Sign up") %>
-<h1><%= t ".signup"%></h1>
-<p><%= t ".str"%></p>
+<h1>Users#new</h1>
+<p>Find me in app/views/users/new.html.erb</p>
```
Comment (thangtx88) [inline app/views/users/new.html.erb]: gì thế này

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -1,3 +1,2 @@
-<% provide(:title, "Sign up") %>
-<h1><%= t ".signup"%></h1>
-<p><%= t ".str"%></p>
+<h1>Users#new</h1>
```
Comment (thangtx88) [inline app/views/users/new.html.erb]: ?????

**Comment by thangtx88** [comment]
Commented

**Comment by hoanganhh** [comment]
ready

**Comment by thangtx88** [comment]
Merged

**Pull request merged**